app/assets/stylesheets/application.scss

```diff
@@ -236,6 +235,7 @@ table {
 
   tr td:last-child {
     .btn {
+      font-size: 13px;
```

Comment:
This makes the edit buttons in forms stick out a bit too much. It sticks out as much as a column title, and more than a primary button text. Also the sub elements on a dropdown menu look weird.

![screenshot from 2015-12-09 10-02-43](https://cloud.githubusercontent.com/assets/598891/11680941/0bfd2cf2-9e5c-11e5-8032-5cb4c0f84f1b.png)

If we respect the default body font 12px for Patternfly and remove this font-size it looks more consistent in my opinion. 

![screenshot from 2015-12-09 10-27-25](https://cloud.githubusercontent.com/assets/598891/11681486/81862674-9e5f-11e5-94e3-4a247cc3d3ef.png)

@kybaker any comments on how the Edit button should look like?